vue.config.js

```diff
@@ -1,3 +1,6 @@
 module.exports = {
   lintOnSave: false,
+
+  pluginOptions: {
+  }
```

Comment:
We can just omit this block since it's now empty

Reply:
It looks like if I omit the key. The lint:style will throw an error `TypeError: Cannot destructure property `lintStyleOnBuild` of 'undefined' or 'null'.
    at module.exports (/home/matthew/girder_web_components/node_modules/@ascendancyy/vue-cli-plugin-stylelint/index.js:7:11)`